### DISCUSSION TITLE
Plugins in Watch mode & with Bundler API

### DISCUSSION THREAD

**Discussion by Devil7DK** [parcel-bundler/parcel#5350] [Q&A] [answered]
Currently plugins are working only with build command. Is there any way to make plugins run in watch mode (or) when using Bundler API...?

**Comment by mischnic** [comment, marked as answer]
Please be more specific. Give an example of a case where it doesn't work as you would expect.

Which Parcel version?

**Reply by Devil7DK** [reply]
Okay. Sorry. My bad.

Those plugins work only on production mode. 😅😅😅
